packages/cli-repl/test/e2e.spec.ts

```diff
@@ -238,5 +238,32 @@ describe('e2e', function() {
       shell.assertNoErrors();
     });
   });
+
+  describe('Ctrl+C aka SIGINT', () => {
+    before(function() {
+      if (process.platform === 'win32') {
```

Comment:
how is it going to behave there?

Reply:
So, it should work there as well, there just isn’t a signal available to us for testing … but I’ll make sure to test this manually on Windows

Reply:
Confirmed that it works there just like it does on other OSes -- and I don't think there's a way to add a test for this, Node.js itself skips this too: https://github.com/nodejs/node/blob/e038d6a1cdb2ffbf666d360553c31786acaf70b2/test/parallel/test-vm-sigint.js#L3-L6